mta-archive-builder/README.md

```diff
@@ -7,11 +7,15 @@ This image can be used to build SAP Multitarget Applications (MTA) containing Ja
 
 On a linux machine you can run 
 
-`docker run -v `pwd`:/project --rm mta-archive-builder mtaBuild --version`
+```
+docker run -v `pwd`:/project --rm mta-archive-builder mtaBuild --version
```

Comment:
Could use `${PWD}` to use the environment variable and not invoke an external command

Reply:
done